.gitignore

```diff
@@ -1,7 +1,3 @@
 /node_modules/
 /typings/
-/jspm_packages/
-/dist/
-npm-debug.log
-.sw*
-.idea
```

Comment:
I don't really mind, but you should create a global gitignore in your computer. If all of us uses 20 different editors / ide, we would need to add them all.

You can go to ~/.gitconfig and do:

```
[core]
	excludesfile = /Users/youruser/.gitignore_global
```

I have something like:

```
.idea
.DS_Store
npm-debug.log
playground
```

So that will get ignored to every project of mine.